jobs/rocoto/aeroanlfinal.sh

```diff
@@ -1,15 +1,21 @@
 #! /usr/bin/env bash
 
-source "$HOMEgfs/ush/preamble.sh"
+source "${HOMEgfs}/ush/preamble.sh"
 
 ###############################################################
-# Source FV3GFS workflow modules
-. $HOMEgfs/ush/load_fv3gfs_modules.sh
+# Source UFSDA workflow modules
+. "${HOMEgfs}/ush/load_ufsda_modules.sh"
 status=$?
-[[ $status -ne 0 ]] && exit $status
+[[ ${status} -ne 0 ]] && exit "${status}"
 
 export job="aeroanlfinal"
 export jobid="${job}.$$"
 
 ###############################################################
-echo "Do nothing for now"
+# setup python path for workflow utilities and tasks
+export PYTHONPATH="$PYTHONPATH:${HOMEgfs}/ush/python:${HOMEgfs}/ush/python/pygw/src"
```

Comment:
```suggestion
pygwPATH="${HOMEgfs}/ush/python:${HOMEgfs}/ush/python/pygw/src"
PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}${pygwPATH}"
export PYTHONPATH
```

Reply:
Is this syntax correct? Perhaps I'm suffering from brain fog but the second `:` is in a weird spot I feel like

Reply:
I believe so.  
What it says is; if `PYTHONPATH` is defined, then add (`+`) it (`PYTHONPATH`) and the `:`  before `pygwPATH`.
If `PYTHONPATH` is not defined, just `pygwPATH`.

Reply:
Yup, Rahul's suggestion is correct.